plugins/catalog-backend/src/database/DefaultProcessingDatabase.ts

```diff
@@ -356,6 +358,18 @@ export class DefaultProcessingDatabase implements ProcessingDatabase {
         this.options.logger.warn(
           `Detected conflicting entityRef ${entityRef} already referenced by ${conflictingKey} and now also ${locationKey}`,
         );
+        if (this.options.conflictEventBroker) {
```

Comment:
Can we just default to a no-op EventBroker?

Reply:
Would that be preferable?